velox/dwio/dwrf/common/FileMetadata.cpp

```diff
@@ -100,11 +101,10 @@ TypeKind TypeWrapper::kind() const {
         return TypeKind::HUGEINT;
       }
     }
-    case proto::orc::Type_Kind_CHAR:
     case proto::orc::Type_Kind_TIMESTAMP_INSTANT:
-      DWIO_RAISE(
+      DWIO_RAISE(fmt::format(
```

Comment:
While you are here, would you also change DWIO_RAISE to VELOX_FAIL?

Reply:
Fixed